### ISSUE TITLE
All Updates and DLC not working

### ISSUE THREAD

**Issue by umbra003** [sudachi-emu/sudachi#86]
I've tried a lot and didn't work...

**Comment by ManuelJimena** [comment]
> I've tried a lot and didn't work... the android version

It started to fail when the switch interface was implemented, just as it happened when it was implemented in Suyu.

**Comment by umbra003** [comment]
yeah it's true when the interface thingy came out there's a lot of negative things going on

**Comment by usaginoshoujo** [comment]
@jarrodnorwell Do you think you could take a look at this? It seems like a high priority issue.

**Comment by ManuelJimena** [comment]
Also, some mods and cheats do not work. For example, in Dark Souls Remastered, the cheat to make the game run at 60 fps does not work, and neither does the mod to fix the black screen problem. All of this works perfectly in v7715077 version.

**Comment by zzzribas** [comment]
Yes, im also getting this problem, if i install manually it works until i close the emulator and open again, i changed my directory of nand can it be it? Does it check when it starts for the default nand and after that unloads the addons or dlcs?
